### PR TITLE
Enable static linking on Windows

### PR DESCRIPTION
**Pull request by nxtn-staged**
Allows use of `FFMPEG_LIBS_DIR` on Windows.

### CODE REVIEW

File: build.rs [hunk 1]
```diff
@@ -342,6 +341,8 @@ fn static_linking(env_vars: &EnvVars) {
                 .expect("Cannot write binding to file.");
         }
```
Comment (ldm0) [inline build.rs:342]: Please add a return here, then the cfg on the panic can be removed.

**Review by ldm0 — CHANGES_REQUESTED** [review]
Thanks! One nitpick:

File: build.rs (hunk 1 above)
Comment (nxtn-staged) [reply build.rs:342]: > the cfg on the panic can be removed

Then you'll get an `unreachable statement` warning as well.

Comment (ldm0) [reply build.rs:342]: fine

**Pull request merged**